adaptation_layer/error_handler.py

```diff
@@ -80,6 +80,11 @@ def __init__(self):
         super().__init__(description='Vim not found.')
 
 
+class NetworkNotFound(Error):
```

Comment:
Maybe SiteNetworkNotFound or VimNetworkNotFound ?

Reply:
Ok, I think VimNetworkNotFound is the best. I'll change it.